src/main/java/com/google/sps/servlets/InfoCardServlet.java

```diff
@@ -76,13 +97,16 @@ public void doPut(HttpServletRequest request, HttpServletResponse response) thro
       response.setStatus(400);
     }
 
+    logger.logp(Level.INFO, SOURCE_CLASS, "doPut", "Got election ID and user address.");
+
     Entity electionEntity = optionalEntity.get();
-    Election election = Election.fromEntity(electionEntity);
 
-    // Don't need to make the API call if this Election object has already been populated.
-    if (election.isPopulatedByVoterInfoQuery()) {
-      return;
-    }
+    // Temporary solution to repeated information in the Datastore.
```

Comment:
Leave a TODO for gianluca, we asked him to write a one pager on this and work on it. Also can you elaborate more on the problem in the comment, pls?